src/plans/CollectSample.plp

```diff
@@ -0,0 +1,26 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Collect a sample from a dug trench, per procedure documented on the
```

Comment:
I don't like having publicly available code point to documentation the public can't see. It's not something we need to fix right now but maybe we can think of a better solution.

Reply:
Agreed.  Fixed - that was simple enough.